File: gestion_covoiturage/src/gestion/livraison/entities/livraison.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package gestion.livraison.entities;

/**
 *
 * @author User */

public class livraison {
    
    private int id_liv , tarif , poids_unitaire ;
    
    private String depart, arrive , type_objet , emplacement_et_etat ;

    public livraison() {
    }

    public livraison( String depart, String arrive,int tarif, String type_objet, String emplacement_et_etat,int poids_unitaire) {
        this.depart = depart;
        this.arrive = arrive;
        this.tarif = tarif;
        this.type_objet = type_objet;
        this.emplacement_et_etat = emplacement_et_etat;
        this.poids_unitaire = poids_unitaire;
    }
    
    public livraison(int id_liv, String depart, String arrive,int tarif, String type_objet, String emplacement_et_etat,int poids_unitaire) {
        this.id_liv = id_liv;
        this.depart = depart;
        this.arrive = arrive;
        this.tarif = tarif;
        this.type_objet = type_objet;
        this.emplacement_et_etat = emplacement_et_etat;
        this.poids_unitaire = poids_unitaire;
    }

    
    
    
    
    
    public int getId_liv() {
        return id_liv;
    }

    public void setId_liv(int id_liv) {
        this.id_liv = id_liv;
    }

    public String getDepart() {
        return depart;
    }

    public void setDepart(String depart) {
        this.depart = depart;
    }

    public String getArrive() {
        return arrive;
    }

    public void setArrive(String arrive) {
        this.arrive = arrive;
    }
    public int getTarif() {
        return tarif;
    }

    public void setTarif(int tarif) {
        this.tarif = tarif;
    }
public String getType_objet() {
        return type_objet;
    }

    public void setType_objet(String type_objet) {
        this.type_objet = type_objet;
    }

public String getEmplacement_et_etat() {
        return emplacement_et_etat;
    }

    public void setEmplacement_et_etat(String emplacement_et_etat) {
        this.emplacement_et_etat = emplacement_et_etat;
    }
     public int getPoids_unitaire() {
        return poids_unitaire;
    }

    public void setPoids_unitaire(int poids_unitaire) {
        this.poids_unitaire = poids_unitaire;
    }
    @Override
    public String toString() {
        return "livraison{" + "id_liv=" + id_liv + ", depart=" + depart + ", arrive=" + arrive +", tarif=" + tarif +", type_objet=" + type_objet +", emplacement_et_etat=" + emplacement_et_etat +", poids_unitaire=" + poids_unitaire + "}\n";
    }
    
    
}

File: gestion_livraison/src/Entity/Livraison.php
<?php

namespace App\Entity;

use App\Repository\LivraisonRepository;
use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;

/**
 * @ORM\Entity(repositoryClass=LivraisonRepository::class)
 */
class Livraison
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(type="integer")
     */
    private $id;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 3,
     *      max = 10,
     *      minMessage = "le depart ne peux pas etre moin de {{ limit }} characters ",
     *      maxMessage = "le depart ne peux pas etre plus de {{ limit }} characters"
     * )
     */
    private $depart;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 3,
     *      max = 10,
     *      minMessage = "le arrivé ne peux pas etre moin de {{ limit }} characters ",
     *      maxMessage = "le arrivé ne peux pas etre plus de {{ limit }} characters"
     * )
     */
    private $arrive;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 3,
     *      max = 10,
     *      minMessage = "le type de l'objet ne peux pas etre moin de {{ limit }} characters ",
     *      maxMessage = "le type de l'objet ne peux pas etre plus de {{ limit }} characters"
     * )
     */
    private $type_objet;

    /**
     * @ORM\Column(type="string", length=255)
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 3,
     *      max = 10,
     *      minMessage = "le emplacement ne peux pas etre moin de {{ limit }} characters ",
     *      maxMessage = "le emplacement ne peux pas etre plus de {{ limit }} characters"
     * )
     */
    private $emplacement_et_etat;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Positive
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 2,
     *      max = 8,
     *      minMessage = "le poids ne peux pas etre moin de {{ limit }} chiffre",
     *      maxMessage = "le poids ne peux pas etre plus de {{ limit }} chiffre"
     * )
     */
    private $poids_unitaire;

    /**
     * @ORM\Column(type="integer")
     * @Assert\Positive
     * @Assert\NotBlank
     * @Assert\Length(
     *      min = 2,
     *      max = 4,
     *      minMessage = "le tarif ne peux pas etre moin de {{ limit }} chiffre",
     *      maxMessage = "le tarif ne peux pas etre plus de {{ limit }} chiffre"
     * )
     */
    private $tarif;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getDepart(): ?string
    {
        return $this->depart;
    }

    public function setDepart(string $depart): self
    {
        $this->depart = $depart;

        return $this;
    }

    public function getArrive(): ?string
    {
        return $this->arrive;
    }

    public function setArrive(string $arrive): self
    {
        $this->arrive = $arrive;

        return $this;
    }

    public function getTypeObjet(): ?string
    {
        return $this->type_objet;
    }

    public function setTypeObjet(string $type_objet): self
    {
        $this->type_objet = $type_objet;

        return $this;
    }
    public function getType_objet(): ?string
    {
        return $this->type_objet;
    }

    public function setType_objet(string $type_objet): self
    {
        $this->type_objet = $type_objet;

        return $this;
    }

    public function getEmplacementEtEtat(): ?string
    {
        return $this->emplacement_et_etat;
    }

    public function setEmplacementEtEtat(string $emplacement_et_etat): self
    {
        $this->emplacement_et_etat = $emplacement_et_etat;

        return $this;
    }
    public function getEmplacement_et_etat(): ?string
    {
        return $this->emplacement_et_etat;
    }

    public function setEmplacement_et_etat(string $emplacement_et_etat): self
    {
        $this->emplacement_et_etat = $emplacement_et_etat;

        return $this;
    }

    public function getPoidsUnitaire(): ?int
    {
        return $this->poids_unitaire;
    }

    public function setPoidsUnitaire(int $poids_unitaire): self
    {
        $this->poids_unitaire = $poids_unitaire;

        return $this;
    }
    public function getPoids_unitaire(): ?int
    {
        return $this->poids_unitaire;
    }

    public function setPoids_unitaire(int $poids_unitaire): self
    {
        $this->poids_unitaire = $poids_unitaire;

        return $this;
    }

    public function getTarif(): ?int
    {
        return $this->tarif;
    }

    public function setTarif(int $tarif): self
    {
        $this->tarif = $tarif;

        return $this;
    }
}
